### PR TITLE
OWA-41: Add manage settings page

### PR DESCRIPTION
**Pull request by jomadoye**
# JIRA TICKET NAME:
OWA-41: [Create Add-on Management setting UI as a migrate from the Legacy UI](https://issues.openmrs.org/browse/OWA-41)

## SUMMARY:
This PR creates a page for the manages OWA settings.

### CODE REVIEW

**Review by wanjikum — COMMENTED** [review]
Good work Jedidah!

**Comment by dkayiwa** [comment]
Thanks @wanjikum for reviewing. 👍

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "wanjikum"
+  ],
+  "license": "MPL-2.0",
+  "keywords": [
+    "OpenMRS",
+    "Open",
+    "Web",
+    "App"
+  ],
+  "homepage": "https://github.com/andela-jomadoye/openmrs-owa-addonmanager",
```
Comment (dkayiwa) [inline bower.json]: Is the above url correct?

File: app/js/components/homePage/manageSettings.jsx [hunk 2]
```diff
@@ -9,11 +9,109 @@
...
+  onChange(event) {
+    event.preventDefault();
+    this.setState({ [event.target.name]: event.target.value });
+  }
+  
+  onCancel() {
+    event.preventDefault();
+    this.setState({ 
+      AppBaseURLValue: '',
+      AppFolderPathValue: 'appdata\\owa',
```
Comment (dkayiwa) [inline app/js/components/homePage/manageSettings.jsx]: Why the backward slashes?

File: app/js/components/homePage/manageSettings.jsx [hunk 3]
```diff
@@ -9,11 +9,109 @@
...
+      AppStoreURLValue: ''
+    };
+
+    this.onChange = this.onChange.bind(this);
+    this.onCancel = this.onCancel.bind(this);
+  }
+
+  componentWillMount() {
+    this.setState({
+      AppFolderPathValue: 'appdata\\owa',
```
Comment (dkayiwa) [inline app/js/components/homePage/manageSettings.jsx]: Why the backward slashes?

File: app/js/components/homePage/manageSettings.jsx (hunk 2 above)
Comment (jomadoye) [reply app/js/components/homePage/manageSettings.jsx]: @dkayiwa  It is only 1 backslash that appears, the second backslash is used to escape it. The final output is `appdata\owa`

File: app/js/components/homePage/manageSettings.jsx (hunk 3 above)
Comment (jomadoye) [reply app/js/components/homePage/manageSettings.jsx]: The double backslash is used to escape the backslash slash.

File: app/js/components/homePage/manageSettings.jsx (hunk 2 above)
Comment (dkayiwa) [reply app/js/components/homePage/manageSettings.jsx]: Why not use a forward slash?

**Comment by jomadoye** [comment]
Thanks @dkayiwa all issues have been fixed.

**Pull request merged**